src/datatable.js

```diff
@@ -334,4 +340,8 @@ export class DataTable {
   displayValue(row, propertyName) {
     return new Homefront(row, Homefront.MODE_NESTED).fetch(propertyName);
   }
+
+  collapseRow(row) {
+    row.collapsed = (row.collapsed) ? false : true;
```

Comment:
To avoid a name collision, it's best to use a Weak-Map and store whether the row is collapsed or not. Another solution would be to follow convention and prepend/append one or two underscores `_collapsed` `__collapsed__`.

I would go for the weakmap solution.

**Read more here.**
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap